scripts/e2e

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+
+
+cd $(dirname $0)/..
+
+if [ ! -f ./bin/terraform-controller ]; then
+    echo "./bin/terraform-controller not built, make build-controller to create."
+    exit 1
+fi
+
+k3s server&
+k3s_pid=$!
+
+kubeconfig=/etc/rancher/k3s/k3s.yaml
+declare -i i; i=0
+until [ -f $kubeconfig ]; do
+    echo "sleeping while waiting for k3s to start..."
+    if (( i > 15 )); then
+        break
+    fi
+
+    sleep 2
+    i+=1
+done
+
+export KUBECONFIG=$kubeconfig
+export NAMESPACE=terraform-controller
+
+./bin/terraform-controller --threads 1&
+tfc_pid=$!
+
+sleep 2 #slight pause to make sure everything is running
+
+echo "Starting e2e Testing"
+
+function cleanup () {
+    echo "Cleaning up k3s ${k3s_pid} and tfc ${tfc_pid}"
+    #kill -9 $k3s_pid
```

Comment:
Commented out for development?

Reply:
this isn't needed any more. forgot to remove.